CHANGES.md

```diff
@@ -70,6 +70,11 @@
 
 * X behavior was changed ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 * A new URN convention was adopted for cross-language transforms and existing URNs were updated. This may break advanced use-cases, for example, if a custom expansion service is used to connect diffrent Beam Java and Python versions. ([BEAM-12047](https://issues.apache.org/jira/browse/BEAM-12047)).
+* We changed the data type for ranges in `JdbcIO.readWithPartitions` from `int` to `long`
+    [BEAM-13149](https://issues.apache.org/jira/browse/BEAM-13149). This is a relatively minor
+    breaking change, which we're implementing to improve the usability of the transform without increasing cruft.
+    This transform is relatively new, so we may implement other breaking changes in the future to improve its usability.
+>>>>>>> a24811cef9 (Documenting change as a breaking change)
```

Comment:
nit: This line seems was added by chance?

Reply:
fixed